databaseManager.js

```diff
@@ -68,25 +68,36 @@ const database = () => {
     const _tagger = getOrCreateUser(tagger);
     const toTag = getOrCreateUser(userToTag, 0, true);
 
-    _tagger.points += 1;
+    const bounty = _calculateBounty(toTag.lastTagged);
+
+    _tagger.points += bounty;
     if (_tagger.isTagged) {
       _tagger.isTagged = false;
     }
 
     // next, reset the tagged user's streak
     const taggedTime = dayjs();
     const streak = taggedTime.diff(toTag.lastTagged, "day");
+    const newPoints = toTag.points + bounty;
     users[userToTag.id] = {
       ...toTag,
       lastTagged: dayjs(),
       isTagged: true,
+      points: newPoints,
       ...{
         bestStreak: streak > toTag.bestStreak ? streak : toTag.bestStreak,
       },
       totalSurvivalTime: toTag.totalSurvivalTime + streak,
     };
     exportDb();
   };
+  const _calculateBounty = (lastTagged, isTagged = false) => {
+    if (isTagged) {
+      return 0;
+    }
+    const now = dayjs();
+    return Math.floor(now.diff(lastTagged, "day", true) * 100);
```

Comment:
Wild inflation compared to the current system. I love it.

Reply:
thought that it would be fun if people are spamming scoreboard or somethign and can see their bounties increasing over the day lol